kinetic/distribution.yaml

```diff
@@ -2828,13 +2828,11 @@ repositories:
       - dynamixel_workbench
       - dynamixel_workbench_controllers
       - dynamixel_workbench_operators
-      - dynamixel_workbench_single_manager
-      - dynamixel_workbench_single_manager_gui
```

Comment:
These two packages will be removed from Kinetic. @ROBOTIS-Will Is this intentional?